README.rst

```diff
@@ -389,6 +389,9 @@ Choose from the list of available fixers:
                 Concatenation should be used with at
                 least one whitespace around.
 
+* **header_comment** [contrib]
+                Add or replace header comment.
```

Comment:
Yes, I wrote this description in my first commit where this operation was not available. I'll update the description.